src/VisualStudio/Xaml/Impl/Features/DocumentSpan.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.Editor.Xaml.Features
+{
+    internal struct DocumentSpan
```

Comment:
Note that @CyrusNajmabadi is also renaming it to `DocumentSpan` in https://github.com/dotnet/roslyn/pull/12979